Jobs/test/Job.IdentityDeletion.Tests/Tests/ActualDeletionWorkerTests.cs

```diff
@@ -91,19 +103,16 @@ private void SetupRipeDeletionProcessesCommand(IMediator mediator, params Identi
         A.CallTo(() => mediator.Send(A<TriggerRipeDeletionProcessesCommand>._, A<CancellationToken>._)).Returns(commandResponse);
     }
 
-    private static ActualDeletionWorker CreateWorker(IMediator mediator, IPushNotificationSender pushNotificationSender)
-    {
-        return CreateWorker(mediator, null, pushNotificationSender);
-    }
-
     private static ActualDeletionWorker CreateWorker(IMediator mediator,
         List<IIdentityDeleter>? identityDeleters = null,
-        IPushNotificationSender? pushNotificationSender = null)
+        IPushNotificationSender? pushNotificationSender = null,
+        IDeletionProcessLogger? deletionProcessLogger = null)
     {
         var hostApplicationLifetime = A.Dummy<IHostApplicationLifetime>();
         identityDeleters ??= [A.Dummy<IIdentityDeleter>()];
         pushNotificationSender ??= A.Dummy<IPushNotificationSender>();
         var logger = A.Dummy<ILogger<ActualDeletionWorker>>();
-        return new ActualDeletionWorker(hostApplicationLifetime, identityDeleters, mediator, pushNotificationSender, logger);
+        deletionProcessLogger ??= A.Dummy<IDeletionProcessLogger>();
```

Comment:
Is it even necessary to pass the IDeletionProcessLogger objects in, like you do in every test?

Reply:
In the `Calls_Deleters_For_Each_Identity` test passing the `IDeletionProcessLogger` is essential to prevent the creation of two distinct instances, which would result in test failure (tested it). However, for the other two tests it is indeed not necessary to pass `IDeletionProcessLogger`. Changes made.